src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -14,6 +14,30 @@
 
     public void moveRobot(Robot robot, int toX, int toY) {
         //your code
+        while (robot.getX() != toX) {
+            if (robot.getX() > toX) {
```

Comment:
try to rotate robot without a loop. you need `while` only for movement, because your robot needs to make some steps in a row, but he doesn't need to find a way he has to go, before every step